.eslintrc.js

```diff
@@ -61,7 +62,10 @@ module.exports = {
     ],
     '@typescript-eslint/no-unused-vars': [
       'error',
-      { ignoreRestSiblings: true },
+      { 
+        ignoreRestSiblings: true,
+        argsIgnorePattern: "^_"
```

Comment:
Why this change?

Reply:
For some of our implementations (i.e macOS, windows), the functions are no-op. The function parameters are unused, so wanted a way to avoid the lint error.

For example, over here: https://github.com/react-native-webview/react-native-webview/blob/ac1b17eaec16af20523d76adf771a4adec2c303d/src/injectJavaScriptWithWebViewKey.macos.ts#L1

`_webViewKey` is not used

Reply:
I see.

1. Do you need the `.macos`/`.windows` versions? I'd expect the generic `injectJavaSCriptWithWebViewKey.ts` to be used in those cases.
2. Is it insufficient to use `// eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars` above the function declaration?

(I don't strongly object to this change, I'm just looking for any opportunities to narrow the scope of what is a very large PR 🙂 )

Reply:
> Do you need the .macos/.windows versions? I'd expect the generic injectJavaScriptWithWebViewKey.ts to be used in those cases.

Ah that's true. I can remove those files and remove this change. 

> Is it insufficient to use // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars above the function declaration? (or I guess just the typescript-eslint one if turning off the other)

We can definitely use those, I figured that there may be a case in the future where you'd want to indicate an unused var without cluttering the code with `// es-lint-disable-next-line`.  

I'll remove these changes to keep the changes smaller for now.

Reply:
I removed the `.eslintrc.js` change here https://github.com/react-native-webview/react-native-webview/pull/2469/commits/4343e47dcc0ca479cb56da74d76c7f904e6131a2 . In a previous commit, I had replaced the `injectJavaScriptWithWebViewKey` files with `WebViewProxy` files. I've also removed the `.macos.ts` and `.windows.ts` files for `WebViewProxy`